src/pages/ProductDetail/ProductBrand/BrandItem/BrandItem.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const BrandItem = ({ title, name, price, img }) => {
+  return (
+    <ItemConatiner>
+      <ItemImg alt={name} src={img} />
+      <ItemTitle>{title}</ItemTitle>
+      <ItemName>{name}</ItemName>
+      <ItemPrice>{`${price.toLocaleString()} 원`}</ItemPrice>
+      <ItemText>즉시 구매가</ItemText>
+    </ItemConatiner>
+  );
+};
+
+const ItemConatiner = styled.li`
```

Comment:
```suggestion
const Conatiner = styled.li`
```
그냥 Container로 해도 괜찮을거 갓습니다